README.md

```diff
@@ -1,17 +1,52 @@
-# eZ Dataflow
+# Code-Rhapsodie EzDataflow Bundle
 
-The eZ Dataflow bundle is intent to manage external data sources import into your EZPlatform instance.
+EzDataflowBundle is a bundle to integrate the Code-Rhapsodie DataflowBundle into eZ Platfom 2.0+.
```

Comment:
`The eZ Dataflow bundle` ... the `Code Rhapsodie Dataflow bundle`.
Are sure we are compatible with Platform 2.0+ ?

Reply:
Compatible with 2.0 ~ 2.5 at least. The admin UI is still the same in 3.0, so it should be okay too.

Reply:
The requirements of the DataflowBundle and the EzDataflowBundle are compatible with the eZ Platform 2.0 requirements.

Reply:
The requirements of the DataflowBundle and the EzDataflowBundle are compatible with eZ Platform 2.0 requirements.